code/modules/antagonists/cult/runes.dm

```diff
@@ -108,7 +108,7 @@ Runes can either be invoked by one's self or with many different cultists. Each
 
 /obj/effect/rune/attack_animal(mob/living/simple_animal/user, list/modifiers)
 	if(isshade(user) || isconstruct(user))
-		if(istype(user, /mob/living/simple_animal/hostile/construct/wraith/angelic) || istype(user, /mob/living/simple_animal/hostile/construct/juggernaut/angelic) || istype(user, /mob/living/simple_animal/hostile/construct/artificer/angelic))
+		if(istype(user, /mob/living/simple_animal/hostile/construct/wraith/angelic) || istype(user, /mob/living/simple_animal/hostile/construct/juggernaut/angelic) || istype(user, /mob/living/basic/construct/artificer/angelic))
```

Comment:
This is an awful check which should be a trait, but you didn't add it so ignore it if you want.